product-service/src/functions/get-product-by-id/handler.ts

```diff
@@ -1,26 +1,23 @@
 import 'source-map-support/register';
-import type { ValidatedEventAPIGatewayProxyEvent } from '@libs/apiGateway';
+import { formatJSONErrorResponse, ValidatedEventAPIGatewayProxyEvent } from '@libs/apiGateway';
 import { formatJSONResponse } from '@libs/apiGateway';
 import { middyfy } from '@libs/lambda';
-import { ProductNotFoundError } from './exceptions';
-import data from 'src/data';
+import ProductNotFoundError from 'src/exceptions/ProductNotFoundException';
+import ProductService from 'src/services/product-service';
 
 const getProductById: ValidatedEventAPIGatewayProxyEvent<any> = async (event) => {
+  console.log(event);
```

Comment:
It's better to add more convenient information for user. It would be hard to understand in CloudWatch what is going on.